queries/atm.py

```diff
@@ -0,0 +1,571 @@
+    dist_sorted = sorted(atms, key=lambda s: s["distance"])
+    return _group_closest_based_on_address(dist_sorted)
+
+
+def _closest_atm_coinmachine(loc: LatLonTuple) -> List[Dict[str, Any]]:
+    """Find ATM closest to the given location that has a coinmachine."""
+    atms: Optional[List[Dict[str, Any]]] = _atms_with_distance(loc)
+    if not atms:
+        return []
+
+    filtered_atms: List[Dict[str, Any]] = []
+    for atm in atms:
+        services = atm.get("services", {})
+        if services.get("coinmachine", False) is True:
+            filtered_atms.append(atm)
+    atms = filtered_atms
+
+    # Sort by distance
+    dist_sorted = sorted(atms, key=lambda s: s["distance"])
+    return _group_closest_based_on_address(dist_sorted)
+
+
+def _format_voice_street_number(s: str) -> str:
+    """
+    Format street name for voice output,
+    also if it contains a range of numbers
+    for example: Fiskislóð 7-9 becomes Fiskislóð sjö til níu
+    """
+    # Formatting numbers
+    word_list = s.split()
+    last_word = word_list[-1]
+    if last_word[-1].isnumeric():
+        if "–" in last_word:
+            # replace irregular hyphen with regular hyphen
+            last_word.replace("–", "-")
+        if "-" in last_word:
+            split_nr = last_word.split("-")
+            first_number = number_to_text(split_nr[0], case="þf")
+            last_number = number_to_text(split_nr[-1], case="þf")
+            number = first_number + " til " + last_number
+            return " ".join(word_list[:-1]) + " " + number
+        return " ".join(word_list[:-1]) + " " + number_to_text(last_word, case="þf")
+    return s
+
+
+def _get_foreign_exchange_string(atm_list: List[Dict[str, Any]]) -> str:
+    """Return a string with foreign exchange info for the given ATM"""
+    atm: Dict[str, Any] = {}
+    # if len(atm_list) == 1:
+    atm = atm_list[0]
+    # else:
+    #    for a in atm_list:
+    #        if a["services"]["foreign_exchange"]["active"]:
+    #            atm = a
+    #            break
+    currency_abr = atm["services"]["foreign_exchange"]["currency"]
+    # Convert currency tags to strings through _FOREIGN_CURRENCY map
+    currencies: List[str] = list()
+    for i in range(len(currency_abr)):
+        print("abr: ", currency_abr[i])
+        currency = NounPhrase(_FOREIGN_CURRENCY[currency_abr[i].strip()]).accusative
+        print("currency: ", currency)
+        if currency is not None:
+            currencies.append(currency)
+    return natlang_seq(currencies)
+
+
+_ERRMSG = "Ekki tókst að sækja upplýsingar um hraðbanka."
+
+
+def _answ_for_atm_query(q: Query, result: Result) -> AnswerTuple:
+    """Return an answer tuple for the given ATM query"""
+    ans_start = ""
+    atm_list: List[Dict[str, Any]] = []
+    if "context_reference" in result and result.last_atm is not None:
+        # There is a reference to a previous result
+        # TODO: Handle multiple ATMs in same location, but with different services
```

Comment:
Could you move the TODO comments to the top of the file, between the module docstring and the imports? Handy for quickly assessing what needs to be done in each query module (usually a lot of things).

Reply:
Fixed.